en_ID/index.html

```diff
@@ -83,6 +83,7 @@
         /* md-text is not of md-col-* but needs the spacing */
         margin-left: 15px;
         margin-right: 15px;
+        color:#8b0000;
```

Comment:
What is this for?

Reply:
oh sorry, i forget to delete this, because i just wanna try it

Reply:
Could you please update?

Reply:
i have done to update, please review